services/fluxmonitor.go

```diff
@@ -171,6 +171,7 @@ func (fm *FluxMonitor) canSubmitUpdated() {
 }
 
 func (fm *FluxMonitor) startTickers() {
+	fm.logger.Info("Starting tickers because node is eligible to submit")
 	fm.tStart.Do(func() {
```

Comment:
```suggestion
	fm.tStart.Do(func() {
		fm.logger.Info("Starting tickers because node is eligible to submit")
```

Do you think it makes more sense to have these inside the function that only runs once? Same applies to stopTickers().

This way we only get log lines when we explicitly start/stop the tickers, instead of each time the function is called. Just seems like it would be easier to trust the logs, but I'd like to hear your opinion too.

Reply:
I see what you mean. This message would be more appropriate inside the running-once functions.

However it does not fulfill one of the reasons added, to have some indicator why the FM is stale, e.g when it starts for the first time and the node is not eligible to submit.

I've split the message in 2 places. What do you think?

Reply:
Sure, that works for me! I'm curious about the FM stale issues - I'll look more into that once I get the next runlog refactoring PR up